src/main/java/school/lemon/changerequest/java/introduction/pr1/Bob.java

```diff
@@ -4,7 +4,25 @@
 public class Bob {
 
     public static String hey(String phrase) {
-        return "";
+        int lengthPhrase = phrase.length();
+        if (lengthPhrase == 0) {
+            return "Fine.";
+        }
+        int number = 0;
+        String newPhrase = phrase.replaceAll("[0-9\\!\\-\\%\\^\\*\\$\\@\\#\\(\\?\\/\\,\\.\\'\\s]", "");
+        String newPhraseNumber = phrase.replaceAll("[\\!\\-\\%\\^\\*\\$\\@\\#\\(\\?\\/\\,\\.\\'\\s]", "");
+        for (int i = 0; i < newPhrase.length() - 1; i++) {
+            if (Character.isUpperCase(newPhrase.charAt(i))) {
+                number++;
+            }
+        }
```

Comment:
This solution do pass all the tests, but it's really to complicated.
As I've mentioned on lecture - if you can solve task without Regexp - solve it without Regexp.
And this task could be solved without Regexp at all.
Try to think about non-regexp solution.